pkg/serverless/trace/propagation/extractor.go

```diff
@@ -112,6 +113,11 @@ func (e Extractor) extract(event interface{}) (*TraceContext, error) {
 		carrier, err = headersCarrier(ev.Headers)
 	case events.LambdaFunctionURLRequest:
 		carrier, err = headersCarrier(ev.Headers)
+	case events.StepFunctionPayload:
+		tc, err := createTraceContextFromStepFunctionInput(ev)
```

Comment:
Can we rename this to `extractTraceContextFromStepFunctionInput` instead? This way it matches the `extractTraceContextfromAWSTraceHeader` above.